src/Libraries/Microsoft.Extensions.AI.Abstractions/Contents/JsonFunctionCallUtilities.cs

```diff
@@ -269,13 +266,18 @@ private static JsonElement GetJsonSchemaCore(JsonSerializerOptions options, Func
                     ? JsonSerializer.Serialize(defaultValue, options.GetTypeInfo(defaultValue.GetType()))
                     : null;
 
-                schemaObj["default"] = defaultValueNode;
+                (schemaObj ??= [])["default"] = defaultValueNode;
             }
 
-            return JsonSerializer.SerializeToElement(schemaObj, FunctionCallHelperContext.Default.JsonNode);
+            return schemaObj is null
+                ? _trueJsonSchema
+                : JsonSerializer.SerializeToElement(schemaObj, FunctionCallUtilityContext.Default.JsonNode);
```

Comment:
It's ok for this to use our built-in converter rather than using the supplied JSO?

Reply:
It's ok, it's just there to convert between the two DOM types. Using the supplied JSO would introduce the implicit dependency for a `JsonNode` contract.